frameworks/javascript/cw-2.js

```diff
@@ -356,6 +356,12 @@ try {
         Test.expect(false, msg, options);
       }
     },
+    assertFuzzyEquals: function (actual, expected, msg = "") {
```

Comment:
We need tests in `test/runners/javascript_spec.js` having a `fixture` allowing inexact solution.